models/engine/db_storage.py

```diff
@@ -74,3 +74,14 @@ def reload(self):
     def close(self):
         """call remove() method on the private session attribute"""
         self.__session.remove()
+
+    def get(self, cls, id):
+        """object based on the class name and its ID, or None"""
+        objs = self.all(cls)
```

Comment:
- Also the naming convention of your variables can be more descriptive, its better to read:

> objects
> results
> data

Instead of

> objs